src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -102,7 +102,7 @@ bool AletheProofPostprocessCallback::update(Node res,
                         << children << " / " << args << std::endl;
 
   NodeManager* nm = nodeManager();
-  std::vector<Node> new_args = std::vector<Node>();
```

Comment:
This should be changed in its own PR if necessary.

Reply:
Removed.